tests/xtimer_now_irq/Makefile.ci

```diff
@@ -0,0 +1,10 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
we should re-check this because the test is actually rather small

Reply:
I checked the list and it seems to fit all these boards just fine - so why are they listed as insufficient memory?

Reply:
I will remove it.